reporter/src/main/kotlin/reporters/AbstractNoticeReporter.kt

```diff
@@ -63,7 +69,10 @@ abstract class AbstractNoticeReporter : Reporter() {
             "This project contains or depends on third-party software components pursuant to the following licenses:\n"
         }
 
-        val noticeReport = NoticeReport(listOf(header), findings, emptyList())
+        val noticeReport = NoticeReport(listOf(header), findings, emptyList()).let { noticeReport ->
+            postProcessingScript?.let { postProcess(ortResult, it, noticeReport) } ?: noticeReport
+        }
```

Comment:
I'm wondering whether it would be nicer to write here

    val noticeReport = postProcess(ortResult, postProcessingScript, NoticeReport(listOf(header), findings, emptyList()))

and handle the case if a `null` `postProcessingScript` inside `postProcess`.